torchgeo/trainers/bigearthnet.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""BigEarthNet trainer."""
+
+from typing import Any, Dict, Optional
+
+import pytorch_lightning as pl
+import torch
+from torch.utils.data import DataLoader
+from torchvision.transforms import Compose
+
+from ..datasets import BigEarthNet
+from ..datasets.utils import dataset_split
+from .tasks import MultiLabelClassificationTask
+
+# https://github.com/pytorch/pytorch/issues/60979
+# https://github.com/pytorch/pytorch/pull/61045
+DataLoader.__module__ = "torch.utils.data"
+
+
+class BigEarthNetClassificationTask(MultiLabelClassificationTask):
```

Comment:
I think getting rid of this class is the last remaining thing to do before this can be merged.

Reply:
Personally I would refactor `ClassificationTask` first in a separate PR to keep this PR from getting any bigger.

Reply:
We should merge this because I still need to refactor the num classes as an input arg before I can remove it. If I do that then I will need to remove/refactor all other classification tasks as well.